### ISSUE TITLE
Container subclass to generate part of my LaTeX report

### ISSUE THREAD

**Issue by martinlarsalbert** [JelteF/PyLaTeX#271]
I have a LaTeX project which consist of many .tex files.
I want to use PyLaTeX to only generate one of these files, so I basically DON'T want PyLaTeX to create the Document etc.

I want something that works like this:
```python
from pylatex.base_classes.containers import Container

class GeneralContainer(Container):
    _latex_name = 'generalcontainer'
    def __init__(self):
        super().__init__()
        
    def dumps(self):
        s = ''
        for child in self:
            if isinstance(child, str):
                s_ = child
            else:
                s_ = child.dumps()
            s+=s_
        return s
```

Usage:
Fill the container with PyLaTeX objects and then save this to a plain .tex file
```python
content= Text()
content.append(...)
content.generate_tex('file.tex')
```
And than I can use this content in whichever way I want in LaTeX
```latex
\section{Section bla bla bla}
\input{file}
```

Basically I need this kind of general container which is already a subclass of Container, does such one exist? Is this a desired use case for PyLaTeX?

**Comment by hallojs** [comment]
I have exactly the same use case and also think that PyLateX could be a suitable tool!

**Comment by JelteF** [comment]
You can probably use the Fragment class that this PR introduces: https://github.com/JelteF/PyLaTeX/pull/292

**Comment by hallojs** [comment]
Thanks @JelteF! Now I'm using the code snippet from above, but I think I can switch when you release the next version of PyLaTeX :)

**Comment by JelteF** [comment]
I just released 1.4.0 with the `Fragment` class included.

**Comment by hallojs** [comment]
Thanks @JelteF!
